server/src-rsr/initialise.sql

```diff
@@ -448,15 +448,22 @@ CREATE VIEW hdb_catalog.hdb_column AS
         , columns.is_nullable
         , columns.ordinal_position
         , coalesce(pkey_refs.ref_tables, '[]') AS primary_key_references
-        , col_description(pg_class.oid, columns.ordinal_position) AS description
+        , col_description(table_oid.oid, columns.ordinal_position) AS description
      FROM information_schema.columns
 LEFT JOIN primary_key_references AS pkey_refs
            ON columns.table_schema = pkey_refs.src_table_schema
           AND columns.table_name   = pkey_refs.src_table_name
           AND columns.column_name  = pkey_refs.src_column_name
-LEFT JOIN pg_class ON pg_class.relname = columns.table_name
-LEFT JOIN pg_namespace ON pg_namespace.oid = pg_class.relnamespace
-          AND pg_namespace.nspname = columns.table_schema;
+LEFT JOIN
+     (
+       SELECT pg_class.oid
+            , pg_class.relname AS table_name
+            , pg_namespace.nspname AS table_schema
+         FROM pg_class
+         LEFT JOIN pg_namespace ON pg_namespace.oid = pg_class.relnamespace
+     ) table_oid ON
+       table_oid.table_name = columns.table_name
+       AND table_oid.table_schema = columns.table_schema;
```

Comment:
Why not just change the original `LEFT JOIN`s to ordinary (`INNER`) `JOIN`s, and move them above the `LEFT JOIN` to `primary_key_references`? That would avoid the issue with a much simpler query, and it’s not like those joins can ever fail, right?

Reply:
Yes, That's much simpler